admin/services/pin.service.js

```diff
@@ -44,8 +44,9 @@ pinService.getActiveSchool = async (schoolId) => {
 pinService.expirePupilPin = async (token, checkCode) => {
   const decoded = jwtService.decode(token)
   const pupil = await pupilDataService.findOne({_id: ObjectId(decoded.sub)})
+  // TODO should this use date service???
   const currentTimeStamp = moment.utc()
-  await checkDataService.update({checkCode: checkCode}, { '$set': { checkStartedAt: currentTimeStamp } })
+  await checkDataService.sqlSetCheckStartedAt(checkCode, currentTimeStamp)
```

Comment:
This is consistent with the previous, so if you change the other this is a reminder for this one.